ouroboros-network/test/Ouroboros/Network/PeerSelection/Test.hs

```diff
@@ -359,8 +359,8 @@ mockPeerSelectionActions' tracer
         -- monitoring loop which does asynchronous demotions. It will terminate
         -- as soon as either of the events:
         --
-        -- * the script returns 'Noop'
-        -- * peer demoted to 'PeerCold'
+        -- + the script returns 'Noop'
+        -- + peer demoted to 'PeerCold'
```

Comment:
I heard there are plans to fix this in haddock :), it will just generate a warning.